### PR TITLE
Refactored About page code

### PR DESCRIPTION
**Pull request by lchua2314**
- Some CSS was not two space indent
- Removed thin border (for now)
- Removed body tag so it doesn't conflict with other pages
- Changed class to className in JSX because it could get confused with classes in JS (convention)
- Changed all em units to rem units to stay consistent

### CODE REVIEW

File: src/modules/About/About.css [hunk 1]
```diff
@@ -20,18 +20,18 @@
...
 .flex-row-container {
-    display: flex;
-    flex-wrap: wrap;
-    align-items: center;
-    justify-content: center;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
 }
```
Comment (roundband5) [inline src/modules/About/About.css:27]: Should add a line space for better readability and to make it more consistent with other elements.
Other than that, the About page looks good.

**Review by roundband5 — CHANGES_REQUESTED** [review]
Add that one line space for better consistency.

File: src/modules/About/About.css (hunk 1 above)
Comment (leonzh2k) [reply src/modules/About/About.css:27]: Follow above advice.

**Review by leonzh2k — APPROVED** [review]

File: src/modules/About/About.css (hunk 1 above)
Comment (lchua2314) [reply src/modules/About/About.css:27]: Done

**Pull request merged**